docs/guides/unbound.md

```diff
@@ -64,6 +64,24 @@ wget -O root.hints https://www.internic.net/domain/named.root
 sudo mv root.hints /var/lib/unbound/
 ```
 
+**Important**: Make sure to give ownership of `root.hints` file to user and group `unbound` by entering:
+
+```bash
+sudo chown unbound:unbound root.hints
```

Comment:
The file is moved before this command. Shouldn't this come first before the `mv` command?